Juriba.DPC/Public/Update-ApplicationOwner.ps1

```diff
@@ -124,81 +124,85 @@ function Update-ApplicationOwner {
         $InputBatchStartOffset += $BatchLength
     } while ($jsonResponse.results -and ($MaximumAppsToImport -eq 0 -or $results.Count -lt $MaximumAppsToImport))
 
-    # Group the results by packageName
-    $groupedResults = $results | Group-Object -Property packageName
+    if ($results.Count -gt 0) {
```

Comment:
do you think that the if {} else{} might be reversed so that the error message is nearer to the check?